core/primitives/src/version.rs

```diff
@@ -236,24 +225,16 @@ impl ProtocolFeature {
             ProtocolFeature::ChunkOnlyProducers | ProtocolFeature::MaxKickoutStake => 56,
             ProtocolFeature::AccountIdInFunctionCallPermission => 57,
 
-            // Nightly & shardnet features, this is to make feature MaxKickoutStake not enabled on
-            // shardnet
+            // Nightly & shardnet features, this is to make feature MaxKickoutStake not enabled on shardnet
+            // TODO which of the features below are nightly and which ones are shardnet and should be deleted?
```

Comment:
what you did is correct.

When we added protocol_feature_fix_staking_threshold - we added it at 126 (which was the nighly version back then) - so that it is not enabled on shardnet (which was on 102 version)